### DISCUSSION TITLE
would this help to run better

### DISCUSSION THREAD

**Discussion by lgwjames** [lllyasviel/Fooocus#2130] [Q&A]
i have a 8gb rx580 in my 32gb mac, but takes 15 mins per pic
been offered this

Nvidia Tesla K10 GPU 8GB GDDR5 3,072 CUDA Cores ** 4.58 Tflops Peak single PFPP

was going to run as a egpu on the thunderport

**Comment by mashb1t** [comment]
GPUs on Mac are not officially supported (no matter if internal or external), but this might still work.

**Comment by Irmagon** [comment]
[SHARK ](https://github.com/nod-ai/SHARK) works fine on AMD vcards.
Witch one little problem - official releases work in WIN system.
Maybe install from sources but work on MAC?

**Comment by lgwjames** [comment]
I'll check that when home 👍

Sent from Outlook for Android<https://aka.ms/AAb9ysg>
________________________________

**Comment by lgwjames** [comment]
https://github.com/nod-ai/SHARK/discussions/2084

tried a few times, same errors
